new-lamassu-admin/src/components/layout/Header.js

```diff
@@ -56,8 +70,31 @@ const Header = memo(({ tree }) => {
     history.push('/maintenance/machine-status', { id: machine.deviceId })
   }
 
+  // these inline styles prevent scroll bubbling: when the user reaches the bottom of the notifications list and keeps scrolling,
+  // the body scrolls, stealing the focus from the notification center, preventing the admin from scrolling the notifications back up
+  // on the first scroll, needing to move the mouse to recapture the focus on the notification center
+  // it also disables the scrollbars caused by the notification center's background to the right of the page, but keeps the scrolling on the body enabled
+  const onClickAway = () => {
+    setAnchorEl(null)
+    document.querySelector('body').removeAttribute('style')
+    document.querySelector('#root').removeAttribute('style')
+  }
+
+  const handleClick = event => {
+    setAnchorEl(anchorEl ? null : event.currentTarget)
+    document.querySelector('body').style.overflow = anchorEl ? 'auto' : 'hidden'
+    const root = document.querySelector('#root')
+    root.style.overflowY = 'scroll'
+    root.style.position = 'absolute'
+    root.style.top = '0'
+    root.style.bottom = '0'
+    root.style.left = '0'
+  }
```

Comment:
Damn, and they're not even handled by react. Here's something you can do:

1- create a class in src/styling/global
2- add and remove the class from the places you need.